opentelemetry-sdk/src/opentelemetry/sdk/_metrics/instrument.py

```diff
@@ -31,18 +31,23 @@
 )
 from opentelemetry._metrics.instrument import UpDownCounter as APIUpDownCounter
 from opentelemetry.sdk._metrics.measurement import Measurement
-from opentelemetry.sdk._metrics.measurement_consumer import MeasurementConsumer
 from opentelemetry.sdk.util.instrumentation import InstrumentationScope
 
-_logger = logging.getLogger(__name__)
+if TYPE_CHECKING:
```

Comment:
What are these type checking changes?

Reply:
They are there to prevent circular import issues caused by the importing of typing symbols, see [here](https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING).